src/api/app/models/kiwi/image.rb

```diff
@@ -103,6 +125,10 @@ def outdated?
 
     package.kiwi_image_outdated?
   end
+
+  def packages
```

Comment:
I think we need to distinguish between packages and kiwi_packages. Because at the moment if you do `kiwi_image.packge` you get a `::Package` but if you do `kiwi_image.packages` you get an array of `Kiwi::Package` objects. So I would rename this method to something else maybe `kiwi_packages`?